src/frontend/src/components/Posts/Posts.jsx

```diff
@@ -26,19 +28,48 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
+const usePrevious = (value) => {
+  const ref = React.useRef(null);
+  React.useEffect(() => {
+    // If data is loaded, store the id of the first post to ref
+    if (value !== undefined) {
+      ref.current = value[0][0].id;
```

Comment:
This will blow up if `value` doesn't exist, or `value[0]` isn't an array.  We should wrap it in a try/catch.